### PR TITLE
:sparkles: Updated.

### PR DESCRIPTION
**Pull request by TheDropX**

### CODE REVIEW

File: websites/P/PornHub/presence.ts [hunk 1]
```diff
@@ -15,22 +16,21 @@ function getTimestamps(
   videoTime: number,
   videoDuration: number
 ): Array<number> {
-  var startTime = Date.now();
-  var endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+  let startTime = Date.now();
+  let endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
```
Comment (ririxichan) [inline websites/P/PornHub/presence.ts:20]: const and merge consts

File: websites/P/PornHub/presence.ts [hunk 2]
```diff
@@ -15,22 +16,21 @@ function getTimestamps(
...
   return [Math.floor(startTime / 1000), endTime];
 }
 
 presence.on("UpdateData", async () => {
   //* If user is on /view_video...
-  var video: HTMLVideoElement = document.querySelector(
+  let video: HTMLVideoElement = document.querySelector(
```
Comment (ririxichan) [inline websites/P/PornHub/presence.ts:26]: let -> const

File: websites/P/PornHub/presence.ts [hunk 3]
```diff
@@ -15,22 +16,21 @@ function getTimestamps(
...
   );
-  if (video[0] !== null && !isNaN(video.duration)) {
+  if (video !== null && !isNaN(video.duration)) {
     //* Get required tags
-    var title: any;
-    title = document.querySelector(".video-wrapper .title-container .title");
+    let title: HTMLElement = document.querySelector(".video-wrapper .title-container .title");
 
-    var uploader = document.querySelector(
+    let uploader: HTMLElement = document.querySelector(
```
Comment (ririxichan) [inline websites/P/PornHub/presence.ts:33]: can be const, and merge them

**Review by ririxichan — CHANGES_REQUESTED** [review]
and upload proof of working presence

File: websites/P/PornHub/presence.ts [hunk 4]
```diff
@@ -15,25 +16,26 @@ function getTimestamps(
...
-        ".video-actions-container .video-info-row .usernameWrap a"
+    const title: HTMLElement = document.querySelector(
+        ".video-wrapper .title-container .title"
       ),
-      timestamps = getTimestamps(
+      uploader: HTMLElement = document.querySelector(
+        ".video-actions-container .video-info-row .usernameWrap a"
+      );
+
+    let timestamps = getTimestamps(
```
Comment (ririxichan) [inline websites/P/PornHub/presence.ts:38]: let -> const

**Review by ririxichan — CHANGES_REQUESTED** [review]

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload proof of it working

**Pull request merged**